packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from "react";
+import filesize from "filesize";
+import { IconNames } from "@jobber/design";
+import styles from "./FormatFile.css";
+import { Button } from "../Button";
+import { Icon } from "../Icon";
+import { Typography } from "../Typography";
+import { ProgressBar } from "../ProgressBar";
+
+interface FileUpload {
```

Comment:
I wonder if we should have a home for types like this that are component agnostic… 🤔

Reply:
For now, I pulled it out into its own file. We can decide where it should live... although a central place makes sense.